src/outdated-snapshot-reporter.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/* eslint-disable class-methods-use-this */
+
+const fs = require('fs');
+const path = require('path');
+
+const TOUCHED_FILE_LIST_PATH = path.join(
+  process.cwd(),
+  '.jest-image-snapshot-touched-files'
+);
+
+const IS_ENABLED = !!process.env.JEST_IMAGE_SNAPSHOT_TRACK_OBSOLETE;
+
+class OutdatedSnapshotReporter {
+  /* istanbul ignore next - test coverage in child process */
+  static isEnabled() {
```

Comment:
Is this used? I assume linting is running on this file can you make sure? I thought it would have caught this...

Reply:
you're right this isn't being used anymore.

linting is running I had to ignore `class-methods-use-this` because to satisfy the jest reporter API